### PR TITLE
reusable utility functions

### LINKED ISSUE #22: add `.py` inside `code/` for reusable fucntions/globals
Add `.py` inside `code/` for reusable fucntions/globals

* from `cityseer.ipynb`
  * `fua_city`
    * ```python
      fua_city = {
           1133: "Aleppo",
           869: "Auckland",
           4617: "Bucaramanga",
           809: "Douala",
           1656: "Liège",
           4881: "Salt Lake City",
       }
       city_fua = {c: f for f, c in fua_city.items()}
       ``` 
    * `read_parquet_roads()`:
     * ```python
         def read_parquet_roads(fua: int) -> geopandas.GeoDataFrame:
              """Read OSM roads from parquet format; return bare columns."""
              return geopandas.read_parquet(pathlib.Path(ddir, f"{fua}", f"roads_osm.{parq}"))[
                  ["highway", "geometry"]
              ].reset_index(drop=True) 
         ```
   * `graph_size()`
      * ```python
         def graph_size(info: str, g: networkx.Graph) -> str:
             return f"{info}\n\t* {g}"
         ```
* update with other notebooks
  * `_clip_networks.ipynb` 
  * `_convert.ipynb` 
  * `cityseer.ipynb` 
  * `clustering.ipynb` 
  * `momepy.ipynb`
  * `osmnx.ipynb` 
  * `usecases.ipynb`

### LINKED ISSUE #36: add wuhan to list of cities
add wuhan to list of cities --> `8989`

* xref https://github.com/uscuni/simplification/pull/29

### PR DESCRIPTION
**Pull request by jGaboardi**
***Please review, but don't merge***

This PR:
* helps get my feet back into things (been neglected this month)
* adds the utility functions mentioned in #22 
* will resolve #22 when complete, but not yet
* if all looks good here, I'll apply these in the other notebooks where possible
* resolves #36

### CODE REVIEW

File: code/utils.py
Comment (martinfleis) [inline code/utils.py:26]: wuhan is missing

**Comment by jGaboardi** [comment]
* **If this current state passes review, let's merge and I will pick back up in another PR**
* @anastassiavybornova
  * I did some streamlining of `osmnx.ipynb` & `usecases.ipynb` (also reran to ensure success)
    * `osmnx.ipynb`
      * general tidying
      * moved video creation into plot loop 
    *  `usecases.ipynb`
       * I added a `load_usecases()` function in `utils.py` for easy access 
       * Also streamlined the structure of the `json`
* @martinfleis 
  * I renamed `cityseer.ipynb` to accommodate yours & mine upcoming exploration. So now we'll have:
     * `cityseer_overview_gaboardi.ipynb`
     * `cityseer_parallel_gaboardi.ipynb` (for my parallel edge exploration)
     * `cityseer_<DESCRIPTION>_fleischmann.ipynb` (for your exploration)

**Review by martinfleis — APPROVED** [review]
fine with me

**Comment by jGaboardi** [comment]
@anastassiavybornova I'll wait to merge until I get you explicit approval since this touches your code. (no pressure/rush, though).

**Review by anastassiavybornova — APPROVED** [review]
looks good to me @jGaboardi !

**Pull request merged**
